src/devices/bus/centronics/ctronics.cpp

```diff
@@ -42,6 +43,11 @@ void centronics_device::device_config_complete()
 	m_dev = dynamic_cast<device_centronics_peripheral_interface *>(get_card_device());
 }
 
+void centronics_device::device_reset()
+{
+	m_sense_handler(m_dev ? 1 : 0);
```

Comment:
I don’t think this is the right thing to do here – devices may or may not actually implement the signal (e.g. a passive R2R DAC in the style of the Covox Speech Thing wouldn’t pull this up, although the Covox implementation used an IBM-style 25-pin connector that lacks this signal).